Framework/TestHelpers/inc/MantidFrameworkTestHelpers/FakeObjects.h

```diff
@@ -149,7 +149,6 @@ class AxeslessWorkspaceTester : public MatrixWorkspace {
       }
       return numY;
     }
-    return m_vec.empty() ? 0 : m_vec[0].dataY().size();
```

Comment:
In addition to this, you can remove the `else` indentation because the `if` statement always returns.
```
if condition {
  return something
}
rest of the function...
```